.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ parameters:
     default: "circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com"
   workbench-image:
     type: string
-    default: "allofustest/workbench:buildimage-0.0.20"
+    default: "allofustest/workbench:gradle7-1"
```

Comment:
This is the only part that disagrees with me. From the Dockerfile, the image we're creating looks more like an "image used to build" than a "image that contains gradle" to me, so I think the name "buildimage" is a closer match than "gradle7."

Reply:
this is the temp image name I think? We probably will use 0.0.22 before merge. I don't have AoU dockerhub permission. @calbach helped me to create a image using this PR, not sure the naming rule here...

Reply:
Yes - this tag was intended as temporary for testing, there are two proposed image changes in flight and I'm not sure which one is being merged first (https://github.com/all-of-us/workbench/pull/5005)

@yonghaoy if this one is going in first, I can relabel accordingly